modules/unsupported/csv/src/test/java/org/geotools/data/csv/CSVDataStoreTest.java

```diff
@@ -159,13 +161,94 @@ public void testFilter() throws FileNotFoundException, IOException {
         SimpleFeatureIterator iter = matches.features();
         while(iter.hasNext()) {
             SimpleFeature f = iter.next();
-            filteredCities.contains(f.getAttribute(CITY_COL));
+            assertTrue(filteredCities.contains(f.getAttribute(CITY_COL)));
             count++;
         }
         iter.close();
         assertEquals(3, count);
         assertEquals(3, matches.size());
     }
+    
```

Comment:
I was unable to grab these three changes as the tests for CSVDataStore have been changed/removed.